src/main/java/cn/nukkit/inventory/DoubleChestInventory.java

```diff
@@ -102,8 +102,6 @@ public void setContents(Map<Integer, Item> items) {
                 } else if (this.right.slots.containsKey(i - this.left.size)) {
                     this.clear(i);
                 }
-            } else if (!this.setItem(i, items.get(i))) {
-                this.clear(i);
```

Comment:
Sorry, these lines can't be deleted

Reply:
yeah that was really fishy. since i'm not used to that api i couldn't review it.